.circleci/config.yml

```diff
@@ -98,6 +98,7 @@ jobs:
                 exit 0
             fi
             demisto-sdk create-id-set -o ./Tests/id_set.json
+            python3 ./Tests/scripts/update_conf_json.py
```

Comment:
Note that if `demisto-sdk create-id-set...` fails then this line will not be called.

Reply:
If the id_set fails then the upload to marketplace/ configure test filter / configure packs to upload will all not work anyway
@glicht

Reply:
@ronykoz  Thinking about the flow here.  I think we should do this step before the id_set step. So when id set runs it has all data and is able to include the relevant tests in its output.